docker/mariadb/00_database-create-geokrety-db.sql

```diff
@@ -219,177 +158,146 @@ CREATE TABLE `gk-miasta` (
 
--- --------------------------------------------------------
-
---
--- Table structure for table `gk-ostatnieruchy`
---
 
 CREATE TABLE `gk-ostatnieruchy` (
-  `ruch_id` int(10) UNSIGNED DEFAULT '0',
-  `id` int(10) UNSIGNED,
+  `ruch_id` int(10) unsigned DEFAULT '0',
+  `id` int(10) unsigned,
   `lat` double(8,5) DEFAULT NULL,
   `lon` double(8,5) DEFAULT NULL,
   `alt` int(5) DEFAULT '-32768',
   `country` varchar(3) CHARACTER SET utf8 COLLATE utf8_polish_ci,
-  `droga` int(10) UNSIGNED,
+  `droga` int(10) unsigned,
   `waypoint` varchar(10) CHARACTER SET utf8 COLLATE utf8_polish_ci,
   `data` datetime DEFAULT NULL,
   `data_dodania` datetime DEFAULT NULL,
-  `user` int(10) UNSIGNED DEFAULT '0',
+  `user` int(10) unsigned DEFAULT '0',
   `koment` varchar(5120) CHARACTER SET utf8 COLLATE utf8_polish_ci DEFAULT NULL,
-  `zdjecia` tinyint(3) UNSIGNED DEFAULT '0',
-  `komentarze` smallint(5) UNSIGNED DEFAULT '0',
+  `zdjecia` tinyint(3) unsigned DEFAULT '0',
+  `komentarze` smallint(5) unsigned DEFAULT '0',
   `logtype` enum('0','1','2','3','4','5','6') CHARACTER SET utf8 COLLATE utf8_polish_ci DEFAULT '0' COMMENT '0=drop, 1=grab, 2=comment, 3=met, 4=arch, 5=dip',
   `username` varchar(20) CHARACTER SET utf8 COLLATE utf8_polish_ci,
   `timestamp` timestamp NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
   `app` varchar(16) CHARACTER SET utf8 COLLATE utf8_polish_ci DEFAULT 'www' COMMENT 'source of the log',
-  `app_ver` varchar(16) CHARACTER SET utf8 COLLATE utf8_polish_ci COMMENT 'application version/codename'
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin;
+  `app_ver` varchar(16) CHARACTER SET utf8 COLLATE utf8_polish_ci COMMENT 'apploction version/codename'
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
 
--- --------------------------------------------------------
-
---
--- Table structure for table `gk-owner-codes`
---
 
 CREATE TABLE `gk-owner-codes` (
-  `id` int(10) UNSIGNED NOT NULL,
-  `kret_id` int(10) UNSIGNED NOT NULL,
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `kret_id` int(10) unsigned NOT NULL,
   `code` varchar(20) COLLATE utf8_polish_ci NOT NULL,
   `generated_date` datetime NOT NULL,
   `claimed_date` datetime NOT NULL,
-  `user_id` int(10) NOT NULL DEFAULT '0'
+  `user_id` int(10) NOT NULL DEFAULT '0',
+  PRIMARY KEY (`id`),
+  KEY `kret_id` (`kret_id`),
+  KEY `code` (`code`)
 ) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_polish_ci;
 
--- --------------------------------------------------------
-
---
--- Table structure for table `gk-races`
---
 
 CREATE TABLE `gk-races` (
-  `raceid` bigint(20) NOT NULL,
+  `raceid` bigint(20) NOT NULL AUTO_INCREMENT,
   `created` datetime NOT NULL COMMENT 'kiedy utworzono',
   `raceOwner` bigint(20) NOT NULL,
   `private` binary(1) NOT NULL DEFAULT '0' COMMENT '0 = public, 1 = private',
-  `haslo` varchar(16) COLLATE utf8_bin NOT NULL COMMENT 'haslo tajnego wyścigu - password for private course',
```

Comment:
Here about comments, still nice to have English comments

Reply:
Fixed